doc/new-guidelines.md

```diff
@@ -784,6 +784,24 @@ src
         └── view.cljs
 ```
 
+### Deprecation process
```

Comment:
The outline level is incorrect. Level 3 puts `Deprecation process` under the 2nd level `Project structure`.